client-mr/pom.xml

```diff
@@ -63,6 +63,11 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
         </dependency>
+        <dependency>
+            <groupId>net.jpountz.lz4</groupId>
+            <artifactId>lz4</artifactId>
+            <version>1.3.0</version>
```

Comment:
dependency should be managed in root pom.xml, and child pom.xml includes it if needed

Reply:
OK.